tracer/test/benchmarks/Benchmarks.OpenTelemetry.InstrumentedApi/Benchmarks.OpenTelemetry.InstrumentedApi.csproj

```diff
@@ -0,0 +1,42 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>net9.0;net8.0;net7.0;net6.0;netcoreapp3.1;net472</TargetFrameworks>
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+    <DefineConstants>INSTRUMENTEDAPI</DefineConstants>
+
+    <RootNamespace>Benchmarks.OpenTelemetry.InstrumentedApi</RootNamespace>
+
+    <!-- benchmarkdotnet only support numberic values, not "latest"-->
+    <LangVersion>11</LangVersion>
```

Comment:
y no 13? 😢

Reply:
Oh I copy/pasted this from the `Benchmark.Trace` project, I can update this later